src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -510,8 +510,8 @@ function ProctoredExamSettings({ courseId, intl }) {
             setCreateZendeskTickets(proctoredExamSettings.create_zendesk_tickets);
           },
         ).catch(
-          errorDetails => {
-            if (errorDetails.customAttributes.httpErrorStatus === 403) {
+          error => {
+            if (error.response.status === 403) {
```

Comment:
The `customAttributes` thing is something we add to our error objects so we can easily log those values to New Relic, as I recall.  `error.response.status` is where axios actually stores the status, and is also compatible with axios-mock-adapter, making it more testable.